ui/src/components/WMWorkerPoolsTable/index.jsx

```diff
@@ -69,49 +69,49 @@ export default class WorkerManagerWorkerTypesTable extends Component {
     this.setState({ sortBy, sortDirection });
   };
 
-  renderRow = workerType => {
+  renderRow = workerPool => {
     const {
       match: { path },
     } = this.props;
     const iconSize = 16;
 
     return (
-      <TableRow key={workerType.workerType}>
+      <TableRow key={workerPool.workerPool}>
```

Comment:
This will end up being `workerPool.workerPoolId` which is a bit easier to interpret (similar to `task.taskId`)

Reply:
oh. So, this PR just a dumb rename. I am currently working on this component displaying actual data - and on that branch, all this stuff is renamed correctly. This is going to cause some conflicts, but it's more convenient for me like that

Reply:
Maybe just close this PR and include the changes with that new PR, then?

Reply:
Or land this -- I guess there's no great harm either way.

Reply:
I saw these as separate tasks, so one PR per each - I agree there shouldn't be harm in doing either way. It's just how I organize my work, that's all